google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/BigtableDataSettings.java

```diff
@@ -193,6 +200,40 @@ public static void enableGfeOpenCensusStats() {
     com.google.cloud.bigtable.data.v2.stub.metrics.RpcViews.registerBigtableClientGfeViews();
   }
 
+  /**
+   * Register built in metrics. This is an experimental feature. Please fill up this form to have
+   * your project allow listed for the private preview: https://forms.gle/xuhu6vCunn2MjV2m9
+   */
+  @BetaApi("Built in metric is not currently stable and may change in the future")
+  public static void registerBuiltinMetrics() throws IOException {
+    GoogleCredentials.getApplicationDefault();
+    if (BUILTIN_METRICS_REGISTERED.compareAndSet(false, true)) {
+      BuiltinViews.registerBigtableBuiltinViews();
+      BigtableStackdriverStatsExporter.register(GoogleCredentials.getApplicationDefault());
+    }
+  }
+
+  /**
+   * Register built in metrics with credentials. This is an experimental feature. Please fill up
+   * this form to have your project allow listed for the private preview:
+   * https://forms.gle/xuhu6vCunn2MjV2m9
+   */
```

Comment:
please add a comment that the credentials must be able to write to all projects that a client will write to